ocaml/parsing/jane_syntax.ml

```diff
@@ -696,9 +699,10 @@ module Layouts = struct
       No_wrap_necessary result_type -> result_type
 
   (*******************************************************)
-  (* Desugaring *)
+  (* Desugaring types *)
 
   let of_type typ =
+    let module Of_ast = Of_ast (Ext) in
```

Comment:
Won't this recreate the module more times than necessary?  I'd hoist it back to the top level